app/lib/snaps/preinstalled/bip32/index.ts

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports, import/no-commonjs*/
+import { assert } from '@metamask/utils';
+const bundleJs = require('../../../../../node_modules/@metamask/bip32-example-snap/dist/bundle.js');
```

Comment:
Is there a reason why these need relative path and can't resolve normally from package names? If so could we leave a comment as to why?
```suggestion
const bundleJs = require('@metamask/bip32-example-snap/dist/bundle.js');
```

(same on line below).

Reply:
Fixed.